packages/ui-components/src/components/PDFExportComponent.tsx

```diff
@@ -14,7 +14,7 @@ export const A4_PAGE_HEIGHT_MM = 297;
  * use absolute length units (like cm and pt) and remove these constants at some point.
  */
 export const A4_PAGE_WIDTH_PX = 1191; // at 144ppi
-export const A4_PAGE_HEIGHT_PX = 1684; // at 144ppi
+export const A4_PAGE_HEIGHT_PX = 1683; // at 144ppi
```

Comment:
This was rounded up from a decimal number before, and this caused landscape pages to overflow vertically